### ISSUE TITLE
"abc.md" has been changed on disk notification, even if it is edited only in marktext

### ISSUE THREAD

**Issue by aisbergde** [marktext/marktext#2170]
Windows version

Today I get sometimes notifications like
> "abc.md" has been changed on disk. Do you want to reload it?

In General this kind of notification is OK. But I get it also, if I work on files *only* in marktext.

If a file is edited only in marktext, then this notification makes not really sense. Or maybe marktext is changing the text itself and then notifies me about changes?

This happens only today, it did not happen yesterday. But I use the same version. Strange

**Comment by fxha** [comment]
> In General this kind of notification is OK. But I get it also, if I work on files only in marktext.

Thank you for reporting the issue. That's very strange and not the expected behavior. Did you opened a file on a network disk or something like that? Please provide some steps how the "bug" occurred. You could try to set `watcherUsePolling` to `true` in `preference.json` located in the [application data directory](https://github.com/marktext/marktext/blob/develop/docs/APPLICATION_DATA_DIRECTORY.md) and try again.

**Comment by aisbergde** [comment]
When I search for watcherUsePolling I see there are several identical issues: #1418 #1261
And yes, my files are in my OneDrive folder (maybe this is the case with many Windows users?)

marktext has a very good documentation, this could be documented there.

**Comment by wernerdaehn** [comment]
I disagree with the label "documentation". It is a bug that should be fixed.
It seems the editor is using an indicator that is changed when the file is synced by onedrive and that does not reliably tell the file was modified.

Having the same problem with onedrive. This message should never popup unless somebody has changed the file while it is opened. That is not the case in this scenario.

**Comment by Jocs** [comment]
Thank you for reporting this! The spurious "file changed on disk" notifications when editing files on OneDrive (or other cloud-synced network drives) are now suppressed in recent builds. We added a content-identical check in PR #4760: when a file-change event fires but the on-disk content is byte-for-byte identical to what is already open in the editor, the notification is silently dropped — so sync-touch events from OneDrive/Dropbox/etc. no longer trigger the dialog.

If you still see unexpected prompts on the latest version with a real content change being involved, please reopen with a repro and the MarkText version number.
